src/functions/model/kontent-client.ts

```diff
@@ -1,12 +1,16 @@
 import { IContentItem, IContentType, DeliveryClient } from '@kontent-ai/delivery-sdk';
 import { KontentConfiguration } from './configuration-model';
+import packageJson from '../../../package.json';
 
 export default class KontentClient {
   client: DeliveryClient
   config: KontentConfiguration
 
   constructor(config: KontentConfiguration) {
-    this.client = new DeliveryClient({ projectId: config.projectId });
+    this.client = new DeliveryClient({
+      projectId: config.projectId,
+      globalHeaders: () => [{ header: 'X-KC-SOURCE', value: `${packageJson.name};${packageJson.version}`}],
```

Comment:
I might add the parameter with underscore to see that original headers are present in `_headers`

Reply:
I am not a fan of adding unused parameters just to see they are there. With TypeScript and modern tools it is not difficult to get the type of the `globalHeaders` property and consequently the type of its argument. (for me it is two clicks or keyboard key presses)